src/components/common/Filter.cy.js

```diff
@@ -13,7 +13,7 @@ describe('Filter', () => {
 
     mount(<Filter onFilter={handleFilter} />);
     cy.contains(/Filters/i).click();
-    cy.contains(/timeline/i);
+    cy.contains(/Planted between/i);
```

Comment:
The integration test was broken.  Seems to be due to this change: https://github.com/Greenstand/treetracker-web-map-client/commit/d765c15f0e02c51e78784e13dc4c8b4803ab1ed7#diff-a0f1006c4f16ceafb5718612eb4c5aae77b963f56857edba07c01c7d78fcaad2L136

Reply:
Yes, we should be more strict on this in the future, the project is getting more complicated.